mix.exs

```diff
@@ -29,14 +29,16 @@ defmodule Sea.MixProject do
   end
 
   def extra_applications(:test), do: [:mox]
+  def extra_applications(:dev), do: [:mox]
```

Comment:
:thinking: 

It won't hurt, but I'm not sure why you did that.

Reply:
Couldn't remember, so I removed it.

Reply:
Now we know:

<img width="1214" alt="Screenshot 2022-11-11 at 19 39 15" src="https://user-images.githubusercontent.com/801356/201408845-bd933c86-2bf5-49e0-a100-3bd871b605cd.png">